src/components/PostsList.tsx

```diff
@@ -1,56 +1,56 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
 import classNames from 'classnames';
 import React from 'react';
 import { Post } from '../types/Post';
+import { useAppDispatch } from '../app/hooks';
+import * as selectedPostActions
+  from '../features/selectedPost/selectedPostSlice';
 
 type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.